### PR TITLE
Add Italian translation

### PR DESCRIPTION
**Pull request by Alexejhero**

### CODE REVIEW

File: Submerged/Localization/Strings/Tasks.it.resx [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+    <data name="PurchaseBreakfast.Offers" xml:space="preserve">
+        <value>OFFERTE:</value>
+    </data>
+    <data name="ReconnectPiping" xml:space="preserve">
+        <value>Riconnetti Tubature</value>
+    </data>
+    <data name="ReconnectPiping.HighPressure" xml:space="preserve">
+        <value>&lt;b&gt;&lt;size=125%&gt;HIGH&lt;/size&gt;
+PRESSIONE&lt;/b&gt;</value>
+    </data>
```
Comment (Alexejhero) [inline Submerged/Localization/Strings/Tasks.it.resx:116]: I don't think this is correct

File: Submerged/Localization/Strings/Tasks.it.resx [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+    <data name="ClearUrchins" xml:space="preserve">
+        <value>Pulisci Animali</value>
+    </data>
+    <data name="CycleReactor" xml:space="preserve">
+        <value>Riattiva Reattore</value>
+    </data>
+    <data name="CycleReactor.RPM" xml:space="preserve">
+        <value>RPM:{0}​</value>
+    </data>
```
Comment (Alexejhero) [inline Submerged/Localization/Strings/Tasks.it.resx:38]: I'm not sure if this is correct

File: Submerged/Localization/Strings/Tasks.it.resx [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+    <data name="MicrowaveLunch" xml:space="preserve">
+        <value>Pranzo Forno A Micronde</value>
+    </data>
+    <data name="MicrowaveLunch.CookTime" xml:space="preserve">
+        <value>Tempo: {0}s
+Nota Per Se:
+
+Prendi Quand'è
+       Pronto</value>
+    </data>
```
Comment (Alexejhero) [inline Submerged/Localization/Strings/Tasks.it.resx:81]: Font doesn't support that è, does e work instead?

Comment (Pietrodjaowjao) [reply Submerged/Localization/Strings/Tasks.it.resx:81]: No

File: Submerged/Localization/Strings/Tasks.it.resx (hunk 1 above)
Comment (Pietrodjaowjao) [reply Submerged/Localization/Strings/Tasks.it.resx:116]: ig he meant ALTA PRESSIONE

File: Submerged/Localization/Strings/Tasks.it.resx (hunk 2 above)
Comment (Pietrodjaowjao) [reply Submerged/Localization/Strings/Tasks.it.resx:38]: It's correct.
image

Comment (Pietrodjaowjao) [reply Submerged/Localization/Strings/Tasks.it.resx:38]: in almost all languagues its written RPM

File: Submerged/Localization/Strings/Tasks.it.resx (hunk 3 above)
Comment (Pietrodjaowjao) [reply Submerged/Localization/Strings/Tasks.it.resx:81]: Maybe it supports "é", that can be used

File: Submerged/Localization/Strings/Tasks.it.resx (hunk 1 above)
Comment (Alexejhero) [reply Submerged/Localization/Strings/Tasks.it.resx:116]:
```suggestion
    <data name="ReconnectPiping.HighPressure" xml:space="preserve">
        <value>&lt;b&gt;&lt;size=125%&gt;ALTA&lt;/size&gt;
PRESSIONE&lt;/b&gt;</value>
    </data>
```

File: Submerged/Localization/Strings/Tasks.it.resx [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+    <data name="SteadyHeartbeat.Target" xml:space="preserve">
+        <value>&lt;u&gt;Note Paziente&lt;/u&gt;
+
+Normal
+Heart Rate:
+{0} - {1} BPM</value>
+    </data>
+    <data name="TrackMantaRay" xml:space="preserve">
+        <value>TracciaManta Ray</value>
+    </data>
```
Comment (Pietrodjaowjao) [inline Submerged/Localization/Strings/Tasks.it.resx:281]: It's missing a space:
"Traccia Manta Ray"

File: Submerged/Localization/Strings/Tasks.it.resx [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+    <data name="SteadyHeartbeat.Target" xml:space="preserve">
+        <value>&lt;u&gt;Note Paziente&lt;/u&gt;
+
+Normal
+Heart Rate:
+{0} - {1} BPM</value>
+    </data>
+    <data name="TrackMantaRay" xml:space="preserve">
+        <value>TracciaManta Ray</value>
```
Comment (Alexejhero) [inline Submerged/Localization/Strings/Tasks.it.resx:280]:
```suggestion
        <value>Traccia Manta Ray</value>
```

**Pull request merged**